### ISSUE TITLE
Accessibility Epic: better contrast in elements in both light and dark modes

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4386]
There are many elements that fail in contrast and the list is described below, mostly taken from the home landing page to indicate initial components and widgets that do repeat across multiple pages.
 
Hopefully this can be a simple change in the CSS files to find new colours that allow for the right level of contrast in both light and dark modes.

### Light mode

Example 1:
![light1a](https://user-images.githubusercontent.com/11466788/137399244-571a0992-df0e-40b4-99d0-54c36f306fc5.png)

Example 2:
![light2a](https://user-images.githubusercontent.com/11466788/137399273-71fcb6f3-e77d-4967-a6a9-7f1fabae3b73.png)

Example 3:
![light3a](https://user-images.githubusercontent.com/11466788/137399306-1c1dc7da-3f3d-4800-aaef-57dafc9b6591.png)

### Dark mode

Example 1:
![dark1](https://user-images.githubusercontent.com/11466788/137399531-e08ea936-1d0a-4921-adb0-15a71c98b708.png)
![dark2](https://user-images.githubusercontent.com/11466788/137399543-7356bc9d-659f-46a9-8660-be64361df4bd.png)

Example 2:
![dark4](https://user-images.githubusercontent.com/11466788/137399580-b861c5cd-4661-4fe7-99cb-90f48fe9446b.png)

Example 3:
Color contrast fails in side nav elements. See _Cloud Credentials_ as an example

### Specific pages

When a new cluster is created, the text is muted and the colour contrast fails:
![cluster1](https://user-images.githubusercontent.com/11466788/137400933-e3b77fc9-65c0-47d3-9c7e-a21b253932a0.png)
Colour contrast fails both the icon and the red text:
![cluster2](https://user-images.githubusercontent.com/11466788/137400943-2a591219-7d8c-4fd6-ae55-fe01fa42f6c6.png)

#### /dashboard/account?sort=age 

Active label fails contrast. The table has 3 different states where the background changes colors. It is important that the colour of the text to have a good contrast (4.5:1) compared against the 3 backgrounds. Also, consider the same check for Dark-theme.
![active1](https://user-images.githubusercontent.com/11466788/137401353-1ee43150-56b1-4907-bdc2-2c49deb1b7fa.png)

ON DIALOG: CHANGE PASSWORD: The blue outlined button to cancel the action fails color contrast heavily when the button is focused. When the button isn’t focused it also fails color contrast criteria.
ON DIALOG: CHANGE PASSWORD: The red button fails color contrast.

#### /dashboard/c/_/manager/provisioning.cattle.io.cluster/create?mode=import

Version text in back left off screen. Color contrast fails 

#### /dashboard/c/local/explorer 

Color contrast fails for the muted text showing the used resources (0.1/4)

#### /dashboard/account/create-key 

Consider improving color contrast for the placeholder text listed in Description
Light mode: Button Fails color contrast. Consider the same recommendations provided above for Icons within buttons with label.
Dark mode: Input fails contrast & label missing  

#### /dashboard/c/local/explorer/tools

Muted text for Istio version fails constrast

#### /dashboard/c/local/explorer/projectsnamespaces

Color contrast in muted text is not enough under _Project: Default_

#### /dashboard/c/local/explorer/workload

Grey buttons contrast issue in dark theme, particularly on hover or click.

#### /dashboard/c/_/manager/provisioning.cattle.io.cluster/create?type=digitalocean

Light theme: 
- colour contrast fails for the red star tied to Cluster Name and then the count corrections (_X 1, 3 or 5_ and _X 1 or more_)
- contrast also fails for _Number of nodes required:_
Dark Theme:
- contrast fails in icon & text for the count corrections, including red & green text

#### /dashboard/c/_/manager/provisioning.cattle.io.cluster/create?type=custom

Light mode:
- _Run this command on one or more existing machines already..._ shows snippet code outside of contrast

**Comment by edenhernandez-suse** [comment]
There are other components with contrast issues: 

1. Links

![image](https://user-images.githubusercontent.com/120640736/211508165-d0c124df-0771-4050-8d78-8080edfb6af7.png)

At 5.37 they pass AA but not AAA, since there are so many links and they are so relevant in the user navigation, I would suggest to try to push it to 7:1 (AAA).


2. Checkboxes / radio buttons

![image](https://user-images.githubusercontent.com/120640736/211508728-d0a2f07b-3842-480e-9f02-e270ec5e32cd.png)

Component borders under 3px width should have a contrast of at least 4.5:1 against the immediate surrounding colour, which does not happen with our checkboxes and radio buttons.


Should we create a checklist with all the detected contrast issues to prioritize them, issue them independently and let developers pick them in order whenever they can? @kwwii @nwmac @gaktive

**Comment by edenhernandez-suse** [comment]
Should we create a checklist with all the detected contrast issues to prioritize them, issue them independently and let developers pick them in order whenever they can? @kwwii @nwmac @gaktive

**Comment by kwwii** [comment]
Yes, we should make a list of specific colors to change. 

Color contrast fails in dark and light theme:
Alerts
Buttons
Forms placeholder text
Badges
Muted text
Links

**Comment by kwwii** [comment]
https://github.com/rancher/dashboard/issues/4380

**Comment by kwwii** [comment]
We may not get all of this epic done but we will start in the upcoming releases
